k8s/k8s.go

```diff
@@ -294,6 +296,45 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, nodename string, calicoCl
 	return result, nil
 }
 
+// ContainerIDMismatchErr is for when ContainerID does not match ActiveInstanceID during Deletion process.
+var ContainerIDMismatchErr = errors.New("ContainerID does not match ActiveInstanceID")
```

Comment:
I think the concept of the Container ID mismatch should be confined to the k8s code.  If you performed the remainder of the k8s WEP delete processing in CmdDelK8s you could achieve that.